build.gradle

```diff
@@ -35,7 +35,8 @@ dependencies {
 	compile gradleApi()
 	compile 'biz.aQute.bnd:bndlib:2.2.0'
 //	compile 'org.osgi:org.osgi.core:4.3.0'
-	compile 'org.eclipse.tycho:org.eclipse.osgi:3.6.2.R36x_v20110210' // provides EclipseEnvironmentInfo
+	compile 'commons-io:commons-io:2.4'
+	compile 'org.eclipse.tycho:org.eclipse.jdt.core:3.11.1.v20150902-1521'
```

Comment:
In `ArtifactFeature.groovy` there is an import: `org.eclipse.jdt.core.dom.ThisExpression`.  The compile fails without this dependency.  It looks to be an unused import to me, but I'm always a little scared to change groovy code, so I just fixed the deps so that it could compile after Gradle 2.14 hid all of its internal deps (thus you have to import your own commons-io, and your own jdt.core).  If you look at the code and agree its an unused import, then we can probably remove it.